src/features/SwapAssets/components/SwapTransactionReceipt.tsx

```diff
@@ -4,11 +4,11 @@ import pick from 'ramda/src/pick';
 
 import { MultiTxReceipt, TxReceipt } from '@components/TransactionFlow';
 import { getFiat } from '@config/fiats';
+import { makeTxItem } from '@helpers';
+import { TxParcel } from '@hooks';
```

Comment:
Shouldn't this be imported from `@types`?

Reply:
Done.